src/containers/App.js

```diff
@@ -1,195 +1,268 @@
-import React, { Component, PropTypes } from 'react';
+import React, {Component, PropTypes} from 'react';
 import fetch from 'isomorphic-fetch';
 import Poster from '../components/Poster';
 import Activities from '../components/Activities';
 import Clock from '../components/Clock';
+// import Quotes from '../components/Quotes';
+import Tile from '../components/Tile';
 
 /**
  * Utility function to change dates from activities to actual Date objects
  */
 function setDate(activity) {
-  return Object.assign(
-    {},
-    activity,
-    { has_start_time : activity.start_date.indexOf('T') > -1 },
-    { has_end_time : activity.end_date && activity.end_date.indexOf('T') > -1 },
-    { start_date: new Date(activity.start_date) },
-    activity.end_date ? { end_date : new Date(activity.end_date) } : null
-  );
+    return Object.assign(
```

Comment:
This reformat makes the diff very hard to read, please do this in a separate commit